### PR TITLE
Repository Maintenance

### PR DESCRIPTION
**Pull request by pietervp**
* Split index.ts into exports (index.ts) and sdk class (sdk.ts)
* Remove obsolete, non functional SDK methods for export
* Synchronise MediaConnector.d.ts type definition with studio-engine

### CODE REVIEW

**Comment by pietervp** [comment]
@brapoprod please review

**Review by abdelhalimkhouas — APPROVED** [review]

**Review by brapoprod — DISMISSED** [review]

**Review by alexandraFlavia9 — DISMISSED** [review]

**Review by brapoprod — APPROVED** [review]

File: types/MediaConnector.d.ts [hunk 1]
```diff
@@ -1,39 +1,44 @@
...
 
-    interface Media {
-        id: String;
-        name: String;
-        relativePath: String;
-        type: number;
-        metaData: Dictionary;
-    }
+export interface Media {
+    id: String;
```
Comment (alexandraFlavia9) [inline types/MediaConnector.d.ts:33]: any particular reason to use String vs string?

Comment (brapoprod) [reply types/MediaConnector.d.ts:33]: goowd one 🦅👁️

Comment (pietervp) [reply types/MediaConnector.d.ts]: Updated the PR to use the Typescript native `string` everywhere in the type definitions (those were the only occurences)

**Review by alexandraFlavia9 — APPROVED** [review]

**Review by brapoprod — APPROVED** [review]

**Pull request merged**